### DISCUSSION TITLE
Installing on a Steam Deck

### DISCUSSION THREAD

**Discussion by LokeYourC3PH** [eza-community/eza#935] [Q&A, Howto, Help]
Hi there! I have been trying to get eza installed on my Steam Deck. Having to get used to Arch Linux is difficult to begin with as is (I can't stand pacman), but whenever I try to install eza via pacman through "pacman -S eza", it simply tells me it cannot find the package. And when I try to do "pacman -S extra/pacman" it tells me the same and that it cannot find the "database". Does anyone have any ideas? I've already disabled read-only mode on my Steam Deck for installing pacman packages, but eza simply can't be found in the extra-3.5 repo.

Thanks in advance!

**Comment by daviessm** [comment]
Hi, you're probably better off asking the Arch people who package it. We just provide the code, they package it.

**Comment by sommerper** [comment]
You could install cargo and install it through that.
